io.github.androidvirtualpen.virtualpen.metainfo.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.androidvirtualpen.virtualpen</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-only</project_license>
+  <name>Android Virtual Pen</name>
+  <summary>Capture android pen input on Linux</summary>
+  <developer id="tld.vendor">
+    <name>Mathieu Beliveau</name>
+  </developer>
+  <description>
+    <p>Android Virtual Pen allows to send pen input data from your android device to a pc host where it emulates a virtual pen. What distinguishes this solution from others, is emphasis on low-latency to provide the best drawing experience.
+    The Android app (.apk) can be found at: https://github.com/androidvirtualpen/virtualpen/releases/download/0.1/virtual-pen.apk</p>
```

Comment:
https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#description

> Descriptions must not contain any direct links.

Reply:
How do you propose that I mention the android's apk? I could do it in the app but seems to me that mentioning it  upfront in the app description would be a good thing, isn't it?

Reply:
Not sure if there's a good way now, you can remove the `https://` so that it doesn't get detected as a link. Links in description aren't clickable anyways.

Reply:
I added a download button within the app, but thx for the suggestion : )